### DISCUSSION TITLE
Github copilot sometimes suggests to import a very huge amount of libaries when told to import a library

### DISCUSSION THREAD

**Discussion by ghost** [community/community#13599] [Copilot Conversations]
![Screenshot from 2022-03-26 23-11-38](https://user-images.githubusercontent.com/78416873/160257279-0c570b73-c91b-4110-9466-04e2115828cc.png)

**Comment by aymenBox** [comment]
Yes with python i noticed that **you cant force it to generate a script for you** , so doing something like **# create an http serve**r
wont actually work or typing **# encrypt a file with a given key** also won't work it won't import one single library to do the work , and it will just keep importing library's at least with python , what you should do instead is to give it context writing those comments are a good thing so he can start to understand what are you trying to do and next what you should do is **Start defining functions** to be used  so instead flowing this practice will improve your experience with copilot , for example if i write a comment **# encrypt a file with a given key** then i start typing def encryptFile it will generate the function for you and import only the wanted library and keep in mind that **copilot won't replace you it's just taking the lead with some instruction**.
